app/src/main/java/org/mozilla/tv/firefox/pocket/PocketRepoCache.kt

```diff
@@ -22,6 +22,7 @@ import io.reactivex.Observable
  * should. This lets us manually control when updates are allowed.
  */
 open class PocketRepoCache(repo: PocketVideoRepo) {
+    // TODO: no longer necessary. Be sure to explain why somewhere.
```

Comment:
Since this isn't required anymore, can we just remove it?  It would make the code less confusing.

Reply:
It'd require some rejiggering of the tests so it didn't seem important, compared to the rest of the changes we needed to get done.